docs/user/dashboard/vega-reference.asciidoc

```diff
@@ -1,8 +1,6 @@
 [[vega-reference]]
 == Vega reference
 
-experimental[]
-
 For additional *Vega* and *Vega-Lite* information, refer to the reference sections.
```

Comment:
This sentence doesn't make sense because we are already looking at the vega reference. I recommend deleting it and the following header.